test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -287,7 +287,8 @@ public void restartTest() throws Exception {
 
         restartServer(corfuRuntime, DEFAULT_ENDPOINT);
 
-        assertThat(corfuRuntime.getLayoutView().getLayout().getEpoch()).isEqualTo(l.getEpoch() + 1);
+        assertThat(corfuRuntime.getLayoutView().getLayout().getEpoch())
+                .isGreaterThanOrEqualTo(l.getEpoch() + 1);
```

Comment:
This doesn't match the comment. Why do you expect the epoch to be greater than or equal, is it because recovery can fail and the epoch can increase multiple times?

Reply:
Yes that's the reason. I'll update the comment to reflect this.